llvm/tools/llvm-cov/CoverageExporterLcov.cpp

```diff
@@ -43,9 +43,26 @@
 #include "CoverageReport.h"
 
 using namespace llvm;
+using namespace coverage;
 
 namespace {
 
+struct NestedCountedRegion : public coverage::CountedRegion {
+  // Contains the path to default and expanded branches
+  // Size is 1 for default branches and greater 1 for expanded branches.
+  std::vector<LineColPair> NestedPath;
+  // Indicates whether this item should at rendering
```

Comment:
Nit: For LLVM, it's typically to end comments with a period.

Reply:
Ah, I didn't know that and updated all my added comments now.